.claude/skills/pr-flow/SKILL.md

```
@@ -1,350 +1,212 @@
+  cron: "*/10 * * * *"
+  prompt: <check-review-prompt>
+  recurring: true
 ```
 
-### Step 3: Wait for CodeRabbit Review
+The check prompt should:
+1. Fetch review status for all tracked PRs via `gh pr view <N> --json reviews`
+2. Check for new inline comments via `gh api repos/{owner}/{repo}/pulls/<N>/comments`
+3. Track consecutive checks with no new activity
+4. After **3 consecutive checks with no CodeRabbit activity**, proactively trigger:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)